src/components/vendor-search-results.js

```diff
@@ -18,7 +18,7 @@ export default function VendorSearchResults({
     <Icon icon="spinner-ellipsis" />
   ) : isRejected ? (
     <p data-test-vendor-search-error-message>
-      {error.length ? error[0].message : error.message}
+      {error.errors.length ? error.errors[0].title : error.title}
```

Comment:
This isn't great, but I figure we can redo how errors are handled in `createRequestReducer()` after we've switched over to 100% JSON API endpoints.

Reply:
Yea, I think we'll have to revisit our request helpers soon anyway (like default JSON serialization/deserialization and better support for updating resources)